starknet_devnet/__init__.py

```diff
@@ -1,14 +1,70 @@
 """
 Contains the server implementation and its utility classes and functions.
 """
+import logging
 import sys
 from copy import copy
+from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple, cast
 
+import cairo_rs_py
 import starkware.cairo.lang.vm.crypto
 from crypto_cpp_py.cpp_bindings import cpp_hash
+from starkware.cairo.common.cairo_function_runner import CairoFunctionRunner
+from starkware.cairo.common.structs import CairoStructFactory, CairoStructProxy
```

Comment:
Couldn't some of these overrides be done on modules instead of individual functions/classes/objects? E.g. instead of overriding `starkware.cairo.common.structs.CairoStructFactory`, couldn't the whole `starkware.cairo` or at least `starkware.cairo.common` be overriden?